zipline/gens/tradesimulation.py

```diff
@@ -356,9 +341,9 @@ def on_dt_changed(self, dt):
         if self.algo.datetime != dt:
             self.algo.on_dt_changed(dt)
 
-    def get_message(self, dt):
+    def generate_messages(self, dt):
         """
-        Get a perf message for the given datetime.
+        Generator that yields perf messages for the given datetime.
```

Comment:
This is a different one than the one we talked about. 

This method used to have clunky logic for handling minute/daily differences, and was reliant on there being exterior logic to create daily perf packets in minutely backtests. I made it a generator because it is where minute and daily perf reporting converge, so by using it as a generator, we know we are getting a stream out regardless of emission rate.